packages/prismic/src/composables/usePrismic/transformQuery.ts

```diff
@@ -43,11 +44,11 @@ const transformQuery = (query: PrismicQuery): string | string[] => {
       default:
         return predict(key, [fragment]);
     }
-  });
+  }));
+
+  const flatArray = [].concat(...queries.filter((queryElement) => queryElement !== undefined));
```

Comment:
`const flatArray = queries.filter((queryElement) => queryElement);`

Reply:
I need to concatenate  it to flatten an array